src/App.vue

```diff
@@ -2,37 +2,107 @@
   <v-app>
     <v-toolbar app>
       <v-toolbar-title class="headline text-uppercase">
-        <span>Vuetify</span>
-        <span class="font-weight-light">MATERIAL DESIGN</span>
+        <span>Posts</span>
       </v-toolbar-title>
       <v-spacer />
-      <v-btn
-        flat
-        href="https://github.com/vuetifyjs/vuetify/releases/latest"
-        target="_blank"
-      >
-        <span class="mr-2">Latest Release</span>
-      </v-btn>
     </v-toolbar>
-
     <v-content>
-      <HelloWorld />
+      <v-card>
+        <v-container
+          fluid
+          grid-list-lg
+        >
+          <v-layout>
+            <v-flex>
+              <v-text-field
+                v-model="searchText"
+                label="Post Id"
+                single-line
+                outline
+              />           
+            </v-flex>
+            <v-flex> 
+              <v-btn
+                large 
+                color="primary"
+                @click="loadPost"
+              >
+                Load
+              </v-btn>
+            </v-flex>
+          </v-layout>
+          <v-layout 
+            v-for="(post, index) in posts"
+            :key="index"
+          >
+            <v-flex xs12>
```

Comment:
Ideally, this v-flex should be a "post" component.

The idea behind vue is to have as many small reusable components as possible.

For instance, this post functionality can be used in other places in the application and therefore makes a good candidate for a component. It will also be the parent component for the "comment" component.